src/ompl/geometric/planners/rrt/RRTstar.h

```diff
@@ -256,14 +322,26 @@ namespace ompl
             /** \brief Deletes (frees memory) the motion and its children motions. */
             void deleteBranch(Motion *motion);
 
-            /** \brief Computes the Cost To Go heuristically as the cost to come from start to motion plus
-                 the cost to go from motion to goal. If \e shortest is true, the estimated cost to come
-                 start-motion is given. Otherwise, this cost to come is the current motion cost. */
-            base::Cost costToGo(const Motion *motion, const bool shortest = true) const;
+            /** \brief Computes the solution cost heuristically as the cost to come from start to the motion plus
+                 the cost to go from the motion to the goal. If \e estimate is true, a heuristic estimate of the
+                 cost to come is used; otherwise, the current cost to come is used. */
+            base::Cost solutionHeuristic(const Motion *motion, const bool estimate = true) const;
```

Comment:
The previous `costToGo` function was poorly named. It calculated more than cost-to-go, but in fact an estimate of an entire solution path constrained to go through that vertex by adding a cost-to-go estimate to the cost-to-come (current or estimated). Renamed as `solutionHeuristic` and cleaned up the documentation.